src/pages/AdminManageAccess.js

```diff
@@ -216,98 +215,91 @@ class AdminManageAccess extends Component {
-                              span({
-                                className: "glyphicon glyphicon-thumbs-up dataset-color",
-                                isRendered: dar.status === 'approved',
-                                "data-tip": "Bonafide researcher",
-                                "data-for": "tip_bonafide"
-                              }),
-                              span({
-                                className: "glyphicon glyphicon-thumbs-down cancel-color",
-                                isRendered: dar.status === 'rejected',
-                                "data-tip": "Non-Bonafide researcher",
-                                "data-for": "tip_nonBonafide"
-                              }),
-                              span({
-                                className: "glyphicon glyphicon-hand-right hover-color",
-                                isRendered: dar.status === 'pending',
-                                "data-tip": "Researcher review pending",
-                                "data-for": "tip_pendingReview"
-                              }),
-                              span({
-                                className: "glyphicon glyphicon-hand-right dismiss-color",
-                                isRendered: dar.status === null
-                              }),
-                            ])
-                        ])
+                    div({ style: {margin: "0 15px 8px"}, className: "bonafide-icon" }, [
+                      a({
+                        id: dar.frontEndId + "_flagBonafide",
+                        name: "flag_bonafide",
+                        onClick: () => this.openResearcherReview('researcher_review', dar.ownerUser.dacUserId)
+                      }, [
+                        span({
+                          className: "glyphicon glyphicon-thumbs-up dataset-color",
+                          isRendered: dar.status === 'approved',
+                          "data-tip": "Bonafide researcher",
+                          "data-for": "tip_bonafide"
+                        }),
+                        span({
+                          className: "glyphicon glyphicon-thumbs-down cancel-color",
+                          isRendered: dar.status === 'rejected',
+                          "data-tip": "Non-Bonafide researcher",
+                          "data-for": "tip_nonBonafide"
+                        }),
+                        span({
+                          className: "glyphicon glyphicon-hand-right hover-color",
+                          isRendered: dar.status === 'pending',
+                          "data-tip": "Researcher review pending",
+                          "data-for": "tip_pendingReview"
+                        }),
+                        span({
+                          className: "glyphicon glyphicon-hand-right dismiss-color",
+                          isRendered: dar.status === null
+                        }),
+                      ])
```

Comment:
lines 226-296 feature no functional changes just formatting to address incorrect indentation warnings